content/news/playtest-20150524.md

```diff
@@ -0,0 +1,55 @@
+It has been a month of busy development. Time to get back to testing.
+
+The mayor changes this time around include:
+
+  - A whole new in-game map editor with all the benefits that come from it.
+  - Improved system integration with `openra://` server join URLs and instant `.orarep` playback.
```

Comment:
You could add a second sentence: "Give it a try from our [live server list](/games/)!".